src/cache/cache.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { CACHE_MANAGER, Controller, Delete, Inject } from '@nestjs/common';
```

Comment:
@hvalette 
the goal of the selective cache management is not to clear the cache with a dedicated endpoint, but to skip the cache middleware and retrieve always the API call selectively with a query parameter.
So the no_cache !== undefined shall skip the middleware and go thru the full API call (for both proxy-api.service and proxy-page.service.
Similarly clear_cache !== undefined shall clear the cache for that concrete key (either api or page) and refresh the value with the response of the Confluence API.

Reply:
I realize that afterward, I'm setting up a cache interceptor for that